docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -271,12 +271,26 @@ You can publish collections to Galaxy using the ``ansible-galaxy collection publ
 
 .. note:: Once you upload a version of a collection, you cannot delete or modify that version. Ensure that everything looks okay before you upload it.
 
+.. _galaxy_get_token:
+
+Getting your token or API key
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To upload your collection to Galaxy, you must first obtain an API token (``--api-key`` in the ``ansible-galaxy`` CLI command). The API token is a secret token used by Ansible Galaxy to protect your content.
+
+To get your API token:
+
+* For galaxy, go to `Galaxy profile preferences <https://galaxy.ansible.com/me/preferences>`_ page and click :guilabel:`API token`.
```

Comment:
For Galaxy, visit the `Galaxy profile preferences <https://galaxy.ansible.com/me/preferences>`_ page and click :guilabel:`API token`.